### DISCUSSION TITLE
Can not work on my SB500

### DISCUSSION THREAD

**Discussion by Footmark0511** [GerryDazoo/Slinger#32] [General]
I tried many times but it still fail. Does someone know what happen on my SB500?

https://imgur.com/a/Ssx376w

http://192.168.0.69:8080/slingbox is not workable with using VLC

**Comment by Footmark0511** [comment]
I thought I need to use SB500 IP for connecting but It does need to use server IP to connect. It is workable now.
Thanks Gerry!
I have another issue now. It is workable on my laptop which is server but not workable on my android phone.
It always shows " Hacker Alert. Invalid Request from  ('192.168.0.235', 46656)" Any suggeston?

https://imgur.com/a/WsnJT1G

**Comment by GerryDazoo** [comment]
Yes. Double check the URL and make sue it says slingbox all lower case.

Or are you trying to use the remote? Make sure it’s Remote  with a capital R.
